src/containers/menus/basemap-selector/styles.module.scss

```diff
@@ -0,0 +1,37 @@
+@import 'styles/settings';
+
+.basemapContainer {
+  bottom: 16px;
+  border: 2px solid $grey-text;
+  border-radius: 4px;
+  display: flex;
+  flex-direction: column;
+  justify-content: end;
+  height: 64px;
+  position: absolute;
+  right: 16px;
+  width: 64px;
+  z-index: $default-z-index;
+  transition: 0.3s;
+  &:hover {
+    border: 2px solid $brand-color-main-hover;
+  }
+
+
+  .basemapLabel {
+    color: $white;
+    font-family: $font-family-1;
+    font-size: 8px;
```

Comment:
Do we have something in typography for this to extend?

Reply:
We dont have one with this fontsize

Reply:
Maybe you can add one then. So if we have this font-size again we will have one

Reply:
Great